github/GithubIntegration.py

```diff
@@ -0,0 +1,188 @@
+import time
+
+import deprecated
+import jwt
+
+from github import Consts
+from github.GithubException import GithubException, UnknownObjectException
+from github.Installation import Installation
+from github.InstallationAuthorization import InstallationAuthorization
+from github.PaginatedList import PaginatedList
+from github.Requester import Requester
+
+
+class GithubIntegration:
+    """
+    Main class to obtain tokens for a GitHub integration.
+    """
+
+    def __init__(
+        self,
+        integration_id,
+        private_key,
+        base_url=Consts.DEFAULT_BASE_URL,
+        jwt_expiry=Consts.JWT_EXPIRY,
+    ):
+        """
+        :param integration_id: int
+        :param private_key: string
+        :param base_url: string
+        :param jwt_expiry: int
+        """
+        assert isinstance(integration_id, (int, str)), integration_id
+        assert isinstance(private_key, str), private_key
```

Comment:
Value of parameter `private_key` should not be leaked, even if it is not a string.
```suggestion
        assert isinstance(private_key, str)
```

Reply:
Replaced by a message not containing the private key